src/sharing/service/SharingServiceClient.ts

```diff
@@ -0,0 +1,34 @@
+import { UserContext } from 'src/auth/dto/UserContext';
+import {
+  ShareTodolistRequest,
+  CheckIfSharedRequest,
+  GetAllSharedTodolistsRequest,
+  GetAllSharedTodolistsResponse,
+} from '../dto/SharingDTO';
+import { SharingServiceInterface } from './SharingServiceInterface';
+import { SharingService } from './SharingService';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class SharingServiceLocalClient implements SharingServiceInterface {
```

Comment:
This should be the only class that is exported (and imported into other services)